demisto_sdk/commands/common/constants.py

```diff
@@ -854,6 +854,7 @@ class PB_Status:
 
 FEED_REQUIRED_PARAMS = [
     {
+        'defaultvalue': 'true',
```

Comment:
shouldn't this be a boolean?

Reply:
In some integration it
defaultvalue: true
and in some its
defaultvalue: 'true'
Both work.
I checked how the majority looks and its that way. defaultvalue: 'true'

Reply:
seems odd that the default value is string when the type of the parameter is boolean but ok...
if that's the majority then let's keep it that way.